ReleaseTooling/Sources/FirebaseManifest/Pod.swift

```diff
@@ -56,12 +56,9 @@ public struct Pod {
   }
 
   /// The Firebase pod does not support import validation with Xcode 12 because of the deprecated
-  /// ML pods not supporting the ARM Mac slice.
+  /// ML pods not supporting the ARM Mac slice. While all other pods support
```

Comment:
Thanks!

The Xcode 12 and ML comment could be removed since it's been a long time since FirebaseML has been part of Firebase.